spec/mongo/collection_spec.rb

```diff
@@ -798,6 +798,29 @@
             expect(change_doc['updateDescription']['updatedFields']['a']).to eq(2)
           end
         end
+
+        context 'when full_document is updateLookup' do
+          before do
+            Thread.new do
+              sleep 1
+              authorized_collection.insert_one(a: 2)
+              authorized_collection.insert_one(a: 3)
+            end
+          end
+
+          let(:change_stream) do
+            authorized_collection.watch([], max_await_time_ms: 1500)
+          end
+
+          it 'returns the documents in the batch size specified' do
```

Comment:
update the context & it labels to refer to this code

Reply:
done